snippets/freeipa_register.erb

```diff
@@ -24,6 +24,9 @@ name: freeipa_register
 #
 #   freeipa_opts                Additional options to pass directly to installer
 #
+
+/usr/sbin/sshd-keygen
```

Comment:
It's certianly available on Jessie, but you'll need to figure out the path:

    greg@topaz:~$ which ssh-keygen
    /usr/bin/ssh-keygen